.github/workflows/note-arduino-ci.yml

```
@@ -0,0 +1,33 @@
+name: Note Arduino CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  validate_examples: # job id
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    strategy:
+      matrix:
+        example-sketch:
+          - ./energy/energy.ino
+        fully-qualified-board-name:
+          - STMicroelectronics:stm32:BluesW:pnum=SWAN_R5
+          # - esp32:esp32:featheresp32
+          # - adafruit:samd:adafruit_feather_m4
```

Should these FQBNs be re-enabled?
```suggestion
          - esp32:esp32:featheresp32
          - adafruit:samd:adafruit_feather_m4
```

---

Thanks, I added these back in.